spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -129,8 +129,8 @@ def resolution_center
         client.get_resolution_center(apple_id, platform)
       end
 
-      def ratings
-        attrs = client.get_rating_summary(apple_id, platform)
+      def ratings(version_id: '', storefront: '')
+        attrs = client.get_rating_summary(apple_id, platform, version_id, storefront)
         attrs[:application] = self
         Tunes::AppRatings.factory(attrs)
```

Comment:
`new attrs` fixed. i don't think i get it 100% what you mean. i thought the ratings is coupled as it requires app_id all-over. 😕